packages/phoenix-evals/src/phoenix/evals/models/openai.py

```diff
@@ -316,18 +337,23 @@ async def _async_generate(self, prompt: Union[str, MultimodalPrompt], **kwargs:
             return str(function_call.get("arguments") or "")
         return str(message["content"])
 
-    def _generate(self, prompt: Union[str, MultimodalPrompt], **kwargs: Any) -> str:
+    def _generate(self, prompt: Union[str | MultimodalPrompt], **kwargs: Any) -> str:
```

Comment:
we can't use `type1 | type2` syntax here, which is equivalent to `Union[type1, type2]`, the original type annotation was correct